src/Alert/Alert.js

```diff
@@ -0,0 +1,54 @@
+import ClassNames from 'classnames'
+import React, { PropTypes } from 'react'
+
+/**
+ * React <b>Alert</b> Component for Patternfly Web Components
```

Comment:
"<b>Alert</b> Component for Patternfly React"

Do you prefer jsdoc style comments? I have recently been looking at [react-docgen](https://github.com/reactjs/react-docgen). If we are interested, I can give this a try. Seems to maybe be a better tool for pure React component docs.

Referencing: [#1](https://github.com/patternfly/patternfly-react/issues/1)